ocaml/xapi/quicktest_cbt.ml

```diff
@@ -60,44 +60,45 @@ let make_vdi_from ~session_id ~sR =
 
 (* Test enable/disable_cbt, data_destroy, and snapshot update the necessary fields *)
 let vdi_data_destroy_test ~session_id ~vDI =
-  let data_destroy_test = make_test "Testing VDI.{enable/disable_cbt, data_destroy, snapshot}" 4 in
+  let test = make_test "Testing VDI.{enable/disable_cbt, data_destroy, snapshot}" 4 in
   try
-    start data_destroy_test;
-    debug data_destroy_test "Enabling CBT on original VDI";
+    start test;
+    let debug_test msg = debug test msg in
```

Comment:
Just wanted to mention that this can be simplified to
```ocaml
let debug_test = debug test in
```
because of higher-order functions. But there is no need to change this.